### PR TITLE
Update the Prerequisites Section in the Getting Started Guides

### LINKED ISSUE #3543: Prerequisite sections of getting started guides are not consistent
**Description:**

Check all the `Prerequisite` sections and have similar wordings, format as most of the time, prerequisite are same.

We have different flavours ATM.

image

image

**Steps to reproduce:**

**Affected Versions:**

**OS, DB, other environment details and versions:**

**Related Issues (optional):**

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Update the Prerequisites section in the Getting Started guides.
> Fixes #3543 

## Check List

- [ ] **Page Addition**
  - [ ] Add `permalink` to pages
  - [ ] If contains empty folder(s), Add front-matter `redirect_to:`

- [ ] **Page Rename**
  - [ ] Add front-matter `redirect_from`
  - [ ] Add front-matter `redirect_to:` (If applicable)

### CODE REVIEW

File: learn/getting-started/deploying-ballerina-on-kubernetes.md [hunk 1]
```diff
@@ -12,15 +12,18 @@ redirect_from:
 
 In this section, you will be writing a simple Ballerina service and then, you will Dockerize the application and deploy it in Kubernetes.
 
-## Prerequisites
+## Setting up the Prerequisites
```
Comment (dakshitha) [inline learn/getting-started/deploying-ballerina-on-kubernetes.md:15]: This is redundant IMO

File: learn/getting-started/working-with-data-in-ballerina.md [hunk 2]
```diff
@@ -10,25 +10,19 @@ redirect_from:
...
-3. Filtering all countries which have more than 10,000,000 Covid cases.
-4. Sorting the top three countries by the number of reported Covid cases.
-5. Joining the table with an array to find the number of recovered patients.
-6. Finding any discrepancies in reported Covid dataset while maintaining intermediate states.
-
-## Prerequisite
+## Setting up the Prerequisite
 
-Following two prerequisites are needed for this tutorial. 
+To complete this tutorial, you need the below.
```
Comment (dakshitha) [inline learn/getting-started/working-with-data-in-ballerina.md:19]: Here we have prerequisites, setting up the prerequisites, and to complete this tutorial, you need the below. Prerequisites alone would suffice.

Comment (praneesha) [reply learn/getting-started/working-with-data-in-ballerina.md:19]: We need to have the `ing` form in the topics and that is why we say `Setting up the Prerequisites` to match with the other titles. 

We can't omit the `to complete this tutorial, you need the below` section as just having the list below the topic would not give the complete meaning that the user needs to set them up as per the topic.

File: learn/getting-started/deploying-ballerina-on-kubernetes.md (hunk 1 above)
Comment (praneesha) [reply learn/getting-started/deploying-ballerina-on-kubernetes.md:15]: Same as this [comment](https://github.com/ballerina-platform/ballerina-dev-website/pull/3564/files#r794989678).

File: learn/getting-started/working-with-data-in-ballerina.md (hunk 2 above)
Comment (dakshitha) [reply learn/getting-started/working-with-data-in-ballerina.md:19]: Let's revise this format in the future then. +1 to go ahead.

**Review by dakshitha — APPROVED** [review]

File: learn/getting-started/deploying-ballerina-on-kubernetes.md [hunk 3]
```diff
@@ -12,15 +12,17 @@ redirect_from:
...
 
-To complete this tutorial, you need:
-
-- A Ballerina installation. Follow the steps in [Installing Ballerina](https://ballerina.io/learn/installing-ballerina/setting-up-ballerina/).
-- Docker installed and configured in your machine.
-- `kubectl` installed and configured to a Kubernetes cluster.
-- Docker hub account
+To complete this tutorial, you need the below.
```
Comment (dakshitha) [inline learn/getting-started/deploying-ballerina-on-kubernetes.md:17]:
```suggestion
To complete this tutorial, you need:```

Comment (praneesha) [reply learn/getting-started/deploying-ballerina-on-kubernetes.md:17]: Fixed. Thanks.

File: learn/getting-started/working-with-data-in-ballerina.md (hunk 2 above)
Comment (praneesha) [reply learn/getting-started/working-with-data-in-ballerina.md:19]: Fixed as per https://github.com/ballerina-platform/ballerina-dev-website/pull/3564#discussion_r794990859.

**Pull request merged**